lib/ReactViews/Custom/Chart/FeatureInfoPanelChart.jsx

```diff
@@ -38,9 +38,14 @@ class FeatureInfoPanelChart extends React.Component {
     );
   }
 
+  componentDidMount() {
+    runInAction(() => {
```

Comment:
This should also run in componentDidUpdate if it’s possible that props.item could change without the component remounting (maybe when there’s multiple feature info results from different items each with a graph? It’s probably still remounting though)